modules/salt/pages/salt-rate-limiting.adoc

```diff
@@ -0,0 +1,54 @@
+[[salt.rate.limiting]]
+= Rate Limiting
+
+Salt is able to run many commands in parallel on a large number of clients.
+This can potentially create large amounts of load on your infrastructure.
+You can use these rate-limiting features to control the load in your environment.
+
+These features are all configured in the [filename]``/etc/rhn/rhn.conf`` configuration file.
+
+
+
+== Batch Size
+
+Actions are processed in batches.
+This parameter sets the maximum number of clients that can execute a single action in each batch.
+
+Choosing a lower number can prevent system thrashing, but might reduce overall performance as parallelization is reduced.
```

Comment:
While the word thrashing probably conveys the meaning intuitively, it has a very specific meaning in jargon and it does not really apply to this case:

https://en.wikipedia.org/wiki/Thrashing_(computer_science)

I'd be happier with something like "Choosing a lower number will reduce the parallelism and load, but potentially also reduces Action processing performance."

Reply:
Hrm, I thought thrashing was a more common way of saying this, I didn't realise how specific it was. I like your suggestion.